portfolio/src/main/webapp/gallery.html

```diff
@@ -1,20 +1,44 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="galleryStyle.css">
     <link rel="stylesheet" href="style.css">
     <title>Gallery</title>
   </head>
   <body>
     <div id="content">
       <h1>Gallery</h1>
-      <p>Coming Soon</p>
     </div>
-
     <div id="sidebar">
       <a href="index.html">Home</a>
       <a href="aboutMe.html">About Me</a>
       <a href="gallery.html">Gallery</a>
     </div>
+    <div class="slideshow">
+      <div class="europeSlideshow">
+        <div class="picCaption">On top of a castle in Ireland</div>
+        <img src="images/slideshowPictures/irelandCastleSQ.JPG" style="width:100%">
+      </div>
+      <div class="europeSlideshow">
+        <div class="picCaption">Out in the gardens of a castle in Scotland</div>
+        <img src="images/slideshowPictures/scotlandCastleSQ.JPG" style="width:100%">
+      </div>
+      <div class="europeSlideshow">
+        <div class="picCaption">Gretna Green in Scotland</div>
+        <img src="images/slideshowPictures/gretnaGreenSQ.JPG" style="width:100%">
+      </div>
+      <div class="europeSlideshow">
+        <div class="picCaption">Out in front of Buckingham Palace</div>
+        <img src="images/slideshowPictures/buckinghamPalaceSQ.JPG" style="width:100%">
+      </div>
+      <div class="europeSlideshow">
+        <div class="picCaption">Out in Spain</div>
+        <img src="images/slideshowPictures/spainJumpingSQ.JPG" style="width:100%">
+      </div>
+      <a class="left-button" onclick="moveSlide(-1);displaySlide();">&#10094;</a>
```

Comment:
nit: you could also just have moveSlide() call displaySlide() itself, so that you don't need to remember to call both